src/dmd/mars.d

```diff
@@ -1508,6 +1525,21 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
             params.map = true;
         else if (arg == "-multiobj")
             params.multiobj = true;
+        else if (startsWith(p + 1, "mixin="))
+        {
+            if (params.mixinFile)
```

Comment:
This follows the -deps example, too, but no other option disallows overriding previous settings.

Reply:
Will fix.